events/testdata/cloudwatch-alarm-sns-payload-single-metric.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Records": [
+        {
+            "EventSource": "aws:sns",
+            "EventVersion": "1.0",
+            "EventSubscriptionArn": "arn:aws:sns:us-east-2: 594917905844:Default_CloudWatch_Alarms_Topic_Test: 872f5b40-db58-4161-94af-4fbc7f613012",
```

Comment:
you should go through these examples and sanitize, they contain information about your aws account and resources

Reply:
Thank you for pointing this out! 

Sanitized the example json payloads: 9f1471ffe8842e044e0261a043db109c233f1d1e